examples/volumes/vsphere/README.md

```diff
@@ -353,6 +353,200 @@
       pvpod       1/1     Running   0          48m        
       ```
 
```

Comment:
We write into kubernetes.io once this PR is merged. Without this PR merged it will be a confusion for the user.